packages/roosterjs-editor-types/lib/interface/TableFormat.ts

```diff
@@ -1,17 +1,13 @@
+import { TableBorderFormat } from '../enum/TableBorderFormat';
+
 /**
  * Table format
  */
 export default interface TableFormat {
-    /**
-     * Background color for even rows
-     */
-    bgColorEven: string;
-
     /**
      * Background color for odd rows
      */
-    bgColorOdd: string;
-
+    bgColor: string | null;

```

Comment:
Comment doesn't match.

Question: Since we already have bgColorOdd and bgColorEven, why need this bgColor? They can be conflict